src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -542,6 +543,11 @@
         }
     }
 
+    .datagrid-foot-description {
+        display: block;
+        white-space: nowrap;
```

Comment:
Yes we should. Before the text was being hidden because I think the height of the footer is fixed? Should I check with @reddolan on what he wants to do here? May be `text-overflow: ellipsis`?